Dockerfile

```diff
@@ -2,8 +2,12 @@
 FROM python:3.8-buster as eventstream
 
 WORKDIR /app
+ARG REQUIREMENTS_FILE
+ENV REQUIREMENTS_FILE=${REQUIREMENTS_FILE:-django.txt}
 COPY requirements/django.txt /app/
-RUN pip install -r django.txt
+COPY requirements/local.txt /app/
```

Comment:
Both requirements files can be copied with a single `COPY` command, reducing the layer count in the image.

Reply:
Done!